README.md

```diff
@@ -758,9 +758,62 @@ state : {
 ```
 
 `Note:`
-values of SkyflowElements will be returned in elementstate object only when `env` is  `DEV`,  else it is empty string i.e, ''
+values of SkyflowElements will be returned in elementstate object only when `env` is  `DEV`,  else it is empty string i.e, '', but in case of card number element when the `env` is `PROD`, it will return first eight digits and rest all digits in masked format.
+
+##### Sample [code snippet](https://github.com/skyflowapi/skyflow-js/blob/master/samples/using-script-tag/collect-element-listeners.html) for using listeners 
+```javascript
+// Create Skyflow client.
+const skyflowClient = Skyflow.init({
+  vaultID: '<VAULT_ID>',
+  vaultURL: '<VAULT_URL>',
+  getBearerToken: () => {},
+  options: {
+    env: Skyflow.Env.DEV,
+  },
+});
+
+const container = skyflowClient.container(Skyflow.ContainerType.COLLECT);
+
+const cardHolderName = container.create({
+  table: 'pii_fields',
+  column: 'first_name',
+  type: Skyflow.ElementType.CARDHOLDER_NAME,
+});
+
```

Comment:
Add CARD_NUMBER Element as well in the sample

Reply:
added